pkg/controller/logstorage/initializer/initializing_controller.go

```diff
@@ -212,9 +212,6 @@ func (r *LogStorageInitializer) Reconcile(ctx context.Context, request reconcile
 		return reconcile.Result{}, err
 	}
 
-	// Determine if Kibana is enabled for this cluster.
-	kibanaEnabled := !operatorv1.IsFIPSModeEnabled(install.FIPSMode) && !r.multiTenant
```

Comment:
I'd prefer to keep `kibanaEnabled := !r.multiTenant` so it is documented by code what we're using r.multiTenant being false means.

Reply:
I addressed this in the code using comments only.